omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/ProviderController1_9Test.java

```diff
@@ -137,6 +137,19 @@ public void shouldFindProviderByUserUuid() throws Exception {
 		Assert.assertEquals(getUuid(), (String) PropertyUtils.getProperty(next, "uuid"));
 	}
 	
+	@Test
+	public void shouldEditAProvider() throws Exception {
+		final String EDITED_PERSON = "da7f524f-27ce-4bb2-86d6-6d1d05312bd5";
+		Provider provider = Context.getProviderService().getAllProviders().get(1);
```

Comment:
Instead of `getAllProviders().get(1)`, why not get a provider with a certain uuid?